samples/CMakeLists.txt

```diff
@@ -59,10 +59,24 @@ if (CONFIG_BT_RPMSG_NRF53)
       "CONFIG_BT_RPMSG_NRF53 is set to 'y'")
 
     if (CONFIG_BT_LL_SOFTDEVICE_DEFAULT)
-      add_overlay_config(
-        hci_rpmsg
-        ${NRF_DIR}/subsys/bluetooth/controller/bt_ll_softdevice_hci_rpmsg.conf
-        )
+      if (EXISTS ${CMAKE_SOURCE_DIR}/bt_ll_softdevice_hci_rpmsg.conf)
+        add_overlay_config(
+          hci_rpmsg
+          ${CMAKE_SOURCE_DIR}/bt_ll_softdevice_hci_rpmsg.conf
+          )
+      else()
+        add_overlay_config(
+          hci_rpmsg
+          ${NRF_DIR}/subsys/bluetooth/controller/bt_ll_softdevice_hci_rpmsg.conf
```

Comment:
Please ensure that this file is added in the same commit to ensure that the commit builds (bisectability etc)